backend/src/modules/tournament-tables/core/domain/TournamentTable.ts

```diff
@@ -0,0 +1,59 @@
+import { EntityIdGenerator } from '../../../../shared/core/application/EntityIdGenerator';
+import { DomainCommandResult } from '../../../../shared/core/domain/DomainCommandResult';
+import { TournamentTablesWereAssigned } from './event/TournamentTablesWereAssigned';
+import { TableNumber } from './TableNumber';
+import { TableId } from './TableId';
+
+export class TournamentTable {
+  readonly tournamentId: string;
+  readonly tableId: TableId;
```

Comment:
Zastnawiam się czy potrzebne nam tableId, skoro mamy tableNumber? Czy nie może być stół jednoznacznie identyfikowany po parze tournamentId i tournamentNumber? No chyba, że to tableId jest już na przyszłośc jako, że ten konkretny stół. Ale to bym rozumiał jakby przychodziło z zewnątrz, a tutaj jest generowane. 
Jeśli w domenie nam to jest po nic (żadna logika nie stoi za idikiem stołu) to można to zostawić dopiero wartstwie bazy danych. Czy jednak bedziemy się gdzieś komunikować tableId zamiast tableNumber?